warp/_src/codegen.py

```
@@ -4230,9 +4353,285 @@ def recurse_subscript(adj, node, indices):
+        except Exception:
+            return None
+
+        bound_arg_types = {k: get_arg_type(v) for k, v in bound_args.items()}
+        bound_arg_values = {k: get_arg_value(v) for k, v in bound_args.items()}
+
+        try:
+            return_type = func.value_func(
+                {k: strip_reference(v) for k, v in bound_arg_types.items()},
+                bound_arg_values,
+            )
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)